_data/sprint_teams.yml

```
@@ -7,6 +7,7 @@
   slack_channel_url: https://gsa-tts.slack.com/archives/CNCGEHG1G
   slack_group_name: login-ada
   readme: https://docs.google.com/document/d/1gAJPduXfsPSXYjlyh_8YxbCtZNONa-jBPYut9T8HKE0/edit
+  product: true
```

Maybe for a follow-on, but might be good to reframe this as one of a few different tokens, and make that field required. Currently, it might just be forgotten to include `product: true` for new product teams.

e.g.

```suggestion
  org_focus: product
```